Feat: list translations command

VasasA opened the pull request:
<!-- FILL OUT ALL RELEVANT SECTIONS, OR THE PULL REQUEST WILL BE CLOSED. -->

## Description

This PR introduces a new command to list translations for Filament 4.x.
"You need to fix the translations where the string has changed. If you want to check the strings, you can show all translations."

## Command Signature
![kép](https://github.com/user-attachments/assets/5a3d5213-2f1f-45e4-a982-3ae3ce3eb3a7)

## Functional changes

Example usage: list filament hungarian translations
```bash
php artisan filament:list-translations hu
```
Command output:
![kép](https://github.com/user-attachments/assets/f0895a05-41a0-41c4-8dd2-f8e8a1912cdd)


- [x] Code style has been fixed by running the `composer cs` command.
- [x] Changes have been tested to not break existing functionality.
- [x] Documentation is up-to-date.

Note: I used the code of '[check-translations](https://github.com/filamentphp/filament/pull/3881)' command.  @saade Thank You!

VasasA committed:
Feat: list translations command

danharrin commented:
I'm sorry I don't really understand what this command would be used for, can you explain please?

VasasA commented:
@danharrin  The "[check-translations](https://github.com/filamentphp/filament/pull/3881)" command helps to find missing and deleted translations. But it does not help to find typos and changed English strings. I need to see all translations (English and Hungarian string pairs) to find all errors.
Please check out [this PR](https://github.com/filamentphp/filament/pull/15074/files) (which I created using the new 'list-translations' command.) The first and second fixes were typos. The third correction fixed a missing string: Two sentences were missing from the Hungarian translation because the English string had been changed earlier.
It is very difficult to check all translations because they are scattered in many language files. This new 'list-translations' command creates an easy-to-read list.
In the future, I will regularly check all Hungarian translations with this command and correct errors by submitting PR-s.

danharrin commented:
Are there ways we can improve the existing command instead of introducing a new one? I am just hesitant to introduce so much new code that very few people will ever use

VasasA commented:
@danharrin I like the idea. I will integrate the listing function into the "check-translations" command, and create a new PR.

danharrin commented:
Thanks. Maybe we can reuse some of the code?

Another piece of feedback - can we use a Laravel Prompts "table" to list instead of the `-->`?

VasasA commented:
I can reuse the code. The integration will be easy.
Using table() function with two columns is not practical because there are some long strings. Some of them can be multiple lines.
